packager/react-packager/src/Resolver/index.js

```diff
@@ -239,4 +241,12 @@ function defineModuleCode(moduleName, code) {
   ].join('');
 }
 
+function definePolyfillCode(code) {
+  return [
+    '(function(global) {\n',
+    `  ${code}`,
```

Comment:
Don’t bother with formatting/indentation, it won’t look good with multiline code anyway :stuck_out_tongue: